lib/daru/view/adapters/googlecharts/display.rb

```diff
@@ -20,9 +21,13 @@ module Display
     # @return [String] The ID of the DIV element that the Google Chart or
     #   Google DataTable should be rendered in
     attr_accessor :html_id
+    attr_accessor :formatters
```

Comment:
Why not in 23rd line itself ?

Reply:
`html_id` is documented on 21st and 22nd lines (above the attribute), so added `attr_accessor :formatters` in a separate line.